pkg/webui/constants/sentry.js

```diff
@@ -0,0 +1,29 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import env from '@ttn-lw/lib/env'
+
+const sentryConfig = {
```

Comment:
Can you also look at adding additional context data: https://docs.sentry.io/enriching-error-data/additional-data/?platform=browser

I think, especially `Sentry.setUser()` would be useful. Based on the action or current location of the user we could set tags like `application`, `gateway,` `end-device`, and even action types such as `get`, `list`, `update`, `delete`, `create` using `Sentry.setTag()`. For TTES we could set tenant and cluster as well.

Reply:
Yes, indeed this would be very nice. I have set the `user_id` tag in the store. I really like the rest of your ideas, however it is a bit more tricky since the `getTag` function of `redux-sentry-middleware` supports passing only `state`, therefore action type or payload will have to be injected some other way in the tags.

I already have ideas for adding action type tags, and tags based on location, but given the priority of this issue I think this should be added in another PR. I suggest we go ahead with this (and it will probably take some time to test that everything even without the tags).